vignettes/datatable-programming.Rmd

```diff
@@ -389,6 +389,29 @@ Users will usually want to substitute the function name rather than inserting th
 
 In case of any doubts on the `env` interface functioning, set `verbose = TRUE` to inspect how expressions are resolved internally.
 
+### Use `env` argument from inside another function
+
+It was a design decision that `env` argument will follow _Standard Evaluation_ rules. As a result, it does not require any special handling when being used from inside another function. As well it does not recognize data.table's `.()` alias for a `list()`.
```

Comment:
is "standard evaluation" a well-defined term? An external reference (e.g. an R manual where this is discussed) would be ideal.

Reply:
precisely NSE is not in R docs, but I think linking this is the best resource on the topic: https://cran.r-project.org/doc/manuals/r-release/R-lang.html#Computing-on-the-language